backend/infrahub/trigger/tasks.py

```diff
@@ -0,0 +1,84 @@
+from typing import TYPE_CHECKING
+
+from infrahub_sdk.utils import compare_lists
+from prefect import get_run_logger, task
+from prefect.automations import AutomationCore
+from prefect.client.orchestration import PrefectClient
+from prefect.client.schemas.filters import DeploymentFilter, DeploymentFilterName
+
+from .catalogue import triggers
+from .constants import DEPRECATED_STATIC_TRIGGER_NAMES
+from .models import TriggerType
+
+if TYPE_CHECKING:
+    from uuid import UUID
+
+
+@task(name="trigger-setup", task_run_name="Setup triggers in task-manager")
+async def setup_triggers(client: PrefectClient) -> None:
```

Comment:
Could we add a docstring here explaining why we are using Prefect triggers? And briefly, how they work, or it might just be a link to corresponding prefect docs.